backend/src/main/java/de/ukrainewin/helpukraine/controller/FoodController.java

```diff
@@ -24,9 +26,9 @@ public class FoodController {
 	}
 
 	@GetMapping("{id}")
-	public /* Food */String listFood(@PathVariable String id) {
+	public Food getFood(@PathVariable String id) {
 		System.out.println("listSuggestedFoodsId");
-		return "one food with id "+ id;// foodService.list();
+		return new Food(id, "some name");// foodService.list();
```

Comment:
Dont return a hard coded object, use your FoodService to recive dynamically the object the user wanted.